_opensearch/remote.md

```diff
@@ -97,21 +97,25 @@ curl -X PUT "https://localhost:9200/my-index?pretty" -ku admin:admin -H 'Content
   "settings": {
     "index": {
       "number_of_shards": 1,
-      "number_of_replicas": 0,
+      "number_of_replicas": 1,
       "replication": {
         "type": "SEGMENT"
       },
       "remote_store": {
         "enabled": true,
-        "repository": "my-repo-1"
+        "repository": "segment-repo",
+        "translog": {
```

Comment:
As `translog` is optional part in `remote_store` setting, we should document the difference between enabling it vs not.

Reply:
Yes, this makes sense. I will make the appropriate changes. Basically, the difference is when you have just the remote_store enabled, it would guarantee refresh level durability, but when translog is enabled as well, it would guarantee request level durability.